BFWControls/Modules/Transition/View/TranslationAnimationController.swift

```diff
@@ -31,12 +31,13 @@ enum Direction: Int {
 
 }
 
-class TranslationAnimationController: NSObject, UIViewControllerAnimatedTransitioning {
+class TranslationAnimationController: UIPercentDrivenInteractiveTransition, UIViewControllerAnimatedTransitioning, UIViewControllerTransitioningDelegate {
+
 
     // MARK: - Variables
 
     @IBInspectable var isPresenting: Bool = true
-    @IBInspectable var duration: TimeInterval = 0.3
+	@IBInspectable var animateDuration: TimeInterval = 0.3
```

Comment:
Fixed in latest commit
- Changed name to transitionDuration and type to CGFloat